README.md

```diff
@@ -140,6 +140,10 @@ You'll need this for your Metricbeat configuration later.
 
 ### System module
 
+**Note**:
+This Docker container monitors Linux system metrics only.
+For other OSes, we recommend running Metricbeat locally on the system itself.
```

Comment:
@imnotashrimp can we add a link to how to do it?

Reply:
@idohalevi You mean... like THIS? 😲

```suggestion
For other OSes, we recommend [running Metricbeat locally](https://docs.logz.io/shipping/metrics-sources/system.html) on the system itself.
```

Reply:
👍